lib/neo4j/active_node/query/query_proxy_link.rb

```diff
@@ -115,7 +115,11 @@ def converted_keys(model, arg)
             end
 
             def converted_key(model, key)
-              key.to_sym == :id ? model.id_property_name : key
+              if key.to_sym == :id
+                model ? model.id_property_name : :uuid
```

Comment:
Haven't actually tested this yet but ❗️ 
I imagine this solution will work for my use case, but won't this fail if someone has defined a custom `id_property`? I don't think there's anyway you can work around that case for everyone, as, as you said, you don't know what the model is so you don't know what the` id_property` is. I think this could be addressed by adding a warning in the docs to users of the custom `id_property` option.

This being said, if someone makes uniform use of a custom `id_property` in their app, then they should be able to use the `pluck(:id)` feature. The docs reference something about setting a custom `id_property` globally (though they don't appear to actually explain how). Maybe, instead of `:uuid`, point to the global config and have the default for that be `:uuid`.

Reply:
Yeah, it could make sense to have a warning both under the documentation for `id_property` as well as the documentation for `model_class`.  It could be that we assume the `model` for the `QueryProxy` is the first model specified and for things like getting the `id_property` both would need to have the same ID property.  But for now I think that this will take care of almost all cases.

I'll add a spec and release it and it could be re-visited later as needed